nsca_client/models/nsca_check.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+# © 2015 ABF OSIELL <http://osiell.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+import shlex
+import subprocess
+
+from openerp import models, fields, api, _
```

Comment:
Please sort all imports in alphabetical order following: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports